pixart.js

```diff
@@ -1 +1,67 @@
+$(document).ready(function (){
 
+// // Put a click event handler on the button Set Color
+// // Get value from input?
+// // Replace background color of brush box with input from form
+$("#set-color-button").on("click", function (event) {
+
+  var firstColor = $("#color-field").eq(0).val();
+
+  var colorBox1 = $("div.colorBox").attr("id", "colorBox1").css({"display": "inline-block", "background-color": firstColor});
+
+  var colorBox2 = $("<div id='colorBox2' class='colorBox'><font color='white'>colorBox2</div>").css({"display": "inline-block", "margin-right": "10px"});
+  event.preventDefault();
+
+  $("#set-color-button").on("click", function (event) {
+
+  var secondColor = $("#color-field").eq(0).val();
+
+  $(colorBox2).css("background-color", secondColor);
+
+  $(colorBox1).css("background-color", firstColor);
```

Comment:
this code will execute synchronously. I think you want this sort of behavior to happen when some kind of event triggers.